src/Service.Host/client/src/components/AllowPurchaseOrderOverbook.js

```diff
@@ -30,7 +30,12 @@ function AllowPurchaseOrderOverbook() {
     const canEdit = () => item?.links.some(l => l.rel === 'allow-over-book');
     const clearErrors = () => reduxDispatch(purchaseOrderActions.clearErrorsForItem());
     const updatePurchaseOrder = () =>
-        reduxDispatch(purchaseOrderActions.update(state.orderNumber, state));
+        reduxDispatch(
+            purchaseOrderActions.update(state.orderNumber, {
+                ...state,
+                currentlyUsingOverbookForm: true
```

Comment:
you (or @RossStewart) should write some react tests to enshrine this as an important part of the request if you do stick with this way of doing it

Reply:
since Ross is gonna patch this won't necessarily apply but remembered I haven't done tests for the reqs form yet :( Will need to come back to that once folk are properly using it and it's not gonna change tons